synthtool/gcp/templates/python_samples/noxfile.py.j2

```diff
@@ -0,0 +1,136 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import os
+from pathlib import Path
+
+import nox
+
+
+# DO NOT EDIT - automatically generated from templates.
+# Edit the `noxfile.py.yml` to change tested versions
+# Current default versions are 3.6, 3.7, and 3.8.
+
+ALL_VERSIONS = [{% for v in DEFAULT_VERSIONS %}"{{v}}"{% if not loop.last %}, {% endif %}{% endfor %}]
+
+# Any default versions that should be ignored.
+IGNORED_VERSIONS = [{% for v in ignored_versions %}"{{v}}"{% if not loop.last %}, {% endif %}{% endfor %}]
+
+TESTED_VERSIONS = sorted([v for v in ALL_VERSIONS if v not in IGNORED_VERSIONS])
```

Comment:
If you can implement `opt_in_sessions:` feature too, that would be great.

Reply:
Should I just copy the `cloud_run` session from the open PR on python-docs-samples?

Reply:
Yeah, that PR successfully run the e2e test, so it's proven to work. I think you'll need to make it no-op by default.